libraries/OPAMP/src/OPAMP.cpp

```diff
@@ -0,0 +1,111 @@
+#include "OPAMP.h"
+#include <Arduino.h>
+
+/* Make sure this library fails to compile for unsupported boards. */
+#if !defined(ARDUINO_UNOWIFIR4) && !defined(ARDUINO_MINIMA)

```

Comment:
```suggestion
#if !defined(ARDUINO_UNOWIFIR4) || !defined(ARDUINO_MINIMA)

```

Reply:
No I don't think that's right. I want the `#error` trigger when it's neither a WiFi R4 nor a Minima. That's !( WIFI || MINIMA)` equivalent `!WIFI && !MINIMA.

Reply:
Agreed, actually. My bad.